syntax = "proto3";

message RespondMsg
{
    bool status = 1;           // 返回值
    int32 seckeyid = 2;    // 对称密钥编号
    bytes clientId = 3;   // 客户端编号
    bytes serverId = 4;   // 服务端编号
    bytes data = 5;       // 服务器端随机数
}

//compile : protoc ./respondmsg.proto --cpp_out=./